openapi/src/main/java/io/micronaut/openapi/annotation/OpenAPIGroup.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2017-2023 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.openapi.annotation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import io.micronaut.context.annotation.AliasFor;
+
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ * With this annotation, you can specify one or more groups that this endpoint will be included in,
+ * as well as specify groups from which this endpoint should be excluded.
+ *
+ * @since 4.9.2
+ */
+@Retention(RUNTIME)
+@Documented
+@Target({ElementType.PACKAGE, ElementType.TYPE, ElementType.METHOD})
+public @interface OpenAPIGroup {
+
+    /**
+     * @return The names of the OpenAPi groups.
+     */
+    @AliasFor(member = "names")
+    String[] value() default {};
+
+    /**
+     * @return The names of the OpenAPi groups.
+     */
+    @AliasFor(member = "value")
+    String[] names() default {};
```

Comment:
I would use one aliasFor depending which value is going to be read

Reply:
I'm sorry, I don't understand what you're suggesting?

Reply:
The is no point to have two `AliasFor`, choose one property that you are going to use for reading (`names`) and only use one alias for `value`

Reply:
It’s not wrong. Essentially you are going to read only one property like names or value, so it doesn’t make sense to map the value to the other value.

Reply:
As far as I understand, AliasFor serves to make the annotations more understandable and at the same time leave the ability to write simple annotations without additional property specification. My case is just right for this:

The OpenAPIGroup annotation has 2 fields - names and exclude.

If you specify both values, then we will write like this: `@OpenAPIGroup(names = "g1", exclude = "g2")`.

But often you do not need to specify the groups from which to exclude this endpoint, then the annotation will look like this:
`@OpenAPIGroup("g1")` . Whihout AliasFor i need to write annotation like this: `@OpenAPIGroup(names = "g1")`